demisto_sdk/commands/common/tools.py

```diff
@@ -414,7 +427,7 @@ def is_origin_content_repo():
     :return: bool : True if remote is configured, False if not.
     """
     remotes = run_command('git remote -v')
-    if re.search(GithubContentConfig().CONTENT_GITHUB_ORIGIN, remotes):
+    if re.search(GitContentConfig().CONTENT_GITHUB_ORIGIN, remotes):
```

Comment:
I know this wasn't written by you, but suggesting changes as this is class field and no need to instantiate an object each time to call them.
```suggestion
    if re.search(GitContentConfig.CONTENT_GITHUB_ORIGIN, remotes):
```